### PR TITLE
Add databag for issues that syncs hourly

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description
This PR adds a hourly cron to sync a databag full of "help wanted" issues to the CCOS site.

## Technical details
This could very quickly cause PRs to CCOS to become outdated by the new commits being pushed every hour.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by zackkrida** [comment]
@dhruvkb I'm missing the context, but is there a reason we should do this instead of loading issues dynamically client-side so they're always up to date by the minute?

**Comment by dhruvkb** [comment]
Yes @zackkrida, loading the issues client-side would have been awesome but GitHub, for some reason, restricts that particular endpoint to [authorised clients only](https://docs.github.com/en/rest/reference/issues#list-organization-issues-assigned-to-the-authenticated-user). The exposed token needs to have `public_repos` scope (which gives it write access) so the only two options are to proxy request to the API via a serverless function or to cache issues hourly (this implementation).

**Comment by zackkrida** [comment]
@dhruvkb Ah yes, that makes perfect sense, thanks. I'm sure in the future we can play with the script frequency or other approaches to keep the data fresher if we feel the need, but an hour seems reasonable right now! Nice work.

**Comment by dhruvkb** [comment]
Please merge creativecommons/creativecommons.github.io-source#510 **before** this.
Please merge creativecommons/creativecommons.github.io-source#502 **after** this.

File: .github/workflows/push_issues_to_ccos.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Push issues to CC Open Source
+on:
+  schedule:
+  - cron: '0 * * * *'
```
Comment (kgodey) [inline .github/workflows/push_issues_to_ccos.yml:4]: I recommend doing this at a random time so that not everything runs at the same time.

**Review by kgodey — CHANGES_REQUESTED** [review]

File: .github/workflows/push_issues_to_ccos.yml (hunk 1 above)
Comment (dhruvkb) [reply .github/workflows/push_issues_to_ccos.yml]: Resolved in 3e2e641.

File: .github/workflows/push_data_to_ccos.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: Push data to CC Open Source
 on: 
   schedule:
-  - cron: '0 0 * * *'
+  - cron: '15 0 * * *'
```
Comment (dhruvkb) [inline .github/workflows/push_data_to_ccos.yml:4]: Scheduled after issues as this is a longer operation.

**Review by kgodey — APPROVED** [review]

**Pull request merged**